### DISCUSSION TITLE
小程序中能用formily/reactive吗

### DISCUSSION THREAD

**Discussion by 1075488066** [alibaba/formily#2763] [Q&A 2.x]
小程序中能用formily/reactive吗

**Comment by 1075488066** [comment]
感觉跨终端的效果没有体现出来。

**Reply by janryWang** [reply]
reactive就是个js库，它的最基本要求就是所在的端支持proxy

**Reply by 1075488066** [reply]
> reactive就是个js库，它的最基本要求就是所在的端支持proxy

官方没有例子，不知道要怎么实现。

**Reply by 1075488066** [reply]
小程序好像没有强制刷新的api。不知道能不能实现
